rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -553,15 +553,3 @@ def get_staking_history(
                     )
                     continue
         return result
-
-    # -- Methods following the EthereumModule interface -- #
-    def on_account_addition(self, address: ChecksumEvmAddress) -> Optional[List['AssetBalance']]:
```

Comment:
In other places where this is done, I see this
```py
def on_account_addition(self, address: ChecksumEvmAddress) -> None:
    pass
```
Any reason that's not done here?

Reply:
Yep, the reason is that this decoder is inherited from `HasDSProxy` which implements this method, so no need to override it here

Reply:
This will be handled anyway via the balance query, later right?

Reply:
Yep, is handled [here](https://github.com/rotki/rotki/blob/develop/rotkehlchen/chain/manager.py#L1280) and [here](https://github.com/rotki/rotki/blob/develop/rotkehlchen/chain/manager.py#L1287)